src/index.html

```diff
@@ -17,6 +17,58 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__icon">
+        <a href="#">
+          <img
+            src="/src/images/logo.png"
+            alt="Moyo Icon"
+            width="40"
+            height="40"
+          />
```

Comment:
Don't use spaces in the `a` tag's `href` property. Specify the link to the home page or the relevant section instead of using `#`.